checks/postgres.go

```diff
@@ -0,0 +1,125 @@
+package checks
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/flanksource/canary-checker/pkg"
+	_ "github.com/lib/pq"
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	postgresFailed = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
Why are these 2 metrics required? Do they not duplicate the standard metrics which provide a duration and failed metric by default ?

Reply:
:man_facepalming: yes!

Reply:
removed with `canary_check_postgres_query_time` metric with 220deb992a75adf5a887bfcc31668cc1417ab26b

Reply:
removed `canary_check_postgres_connectivity_failed` with 76e9128b91c5677bda638fe6717f90b41f290ac8